### PR TITLE
Update v2 README to prefer installation from Helm

### PR DESCRIPTION
**Pull request by matthchr**
Added alternative docs path for installing straight from the release
YAML.

**If applicable**:
- [x] this PR contains documentation

### CODE REVIEW

File: docs/hugo/content/introduction/installing-from-yaml.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+### Prerequisites
+1. You have followed the [installation instructions](https://azure.github.io/azure-service-operator/#installation) up to the "install from Helm" step.
+
+### Installation
+
+1. Install [the latest **v2+** release](https://github.com/Azure/azure-service-operator/releases) of Azure Service Operator.
+   ```bash
+   kubectl apply --server-side=true -f https://github.com/Azure/azure-service-operator/releases/download/v2.0.0-alpha.6/azureserviceoperator_v2.0.0-alpha.6.yaml
+   ```
+2. Create the Azure Service Operator v2 secret. This secret contains the identity that Azure Service Operator will run as. Make sure that you have the 4 environment variables from step 2 set before running this command.
```
Comment (theunrepentantgeek) [inline docs/hugo/content/introduction/installing-from-yaml.md:14]: This _is_ Step 2 - so "... environment variables from step 2 set ..." is confusing to me.

Is this referring back to the installation instructions listed as a prerequisite? If so, perhaps just list having those variables set as a second pre-requ.

File: v2/README.md [hunk 2]
```diff
@@ -68,25 +68,21 @@ See the list of supported resources [here](https://azure.github.io/azure-service
    AZURE_CLIENT_ID=<your-client-id> # This is the appID from the service principal we created.
    AZURE_CLIENT_SECRET=<your-client-secret> # This is the password from the service principal we created.
    ```
-3. Install [the latest **v2+** release](https://github.com/Azure/azure-service-operator/releases) of Azure Service Operator:
-   ```bash
-   kubectl apply --server-side=true -f https://github.com/Azure/azure-service-operator/releases/download/v2.0.0-alpha.6/azureserviceoperator_v2.0.0-alpha.6.yaml
+
+3. Install [the latest **v2+** Helm chart](/v2/charts). Alternative you can install from the [release YAML directly]()
```
Comment (theunrepentantgeek) [inline v2/README.md:72]: Link is missing.

**Review by theunrepentantgeek — CHANGES_REQUESTED** [review]

File: v2/README.md (hunk 2 above)
Comment (matthchr) [reply v2/README.md:72]: Doh, good eye.

Comment (matthchr) [reply v2/README.md:72]: Fixed

File: docs/hugo/content/introduction/installing-from-yaml.md (hunk 1 above)
Comment (matthchr) [reply docs/hugo/content/introduction/installing-from-yaml.md]: Fixed. Hopefully should be clearer now.

**Review by theunrepentantgeek — APPROVED** [review]

**Pull request merged**